### ISSUE TITLE
Replace PlexRequests with Ombi

### ISSUE THREAD

**Issue by NumberedThought** [bytesizedhosting/bcd#35]
Request that the PlexRequests app be replaced with Ombi:  http://www.ombi.io/#Ombi

Ombi Provides a docker image at:  https://github.com/rogueosb/docker-ombi

**Comment by maran** [comment]
No need to replace anything, but I will check this out as I'm not the biggest fan of PlexRequests Meteor.

**Comment by NumberedThought** [comment]
True.  It's a mono app though, and it gets all the baggage of that.

A little bit more feature rich.

**Comment by maran** [comment]
Yeah that's really too bad. Mono is an other thing you would preferably not have on your server at all.
